styles/modules/listing.scss

```diff
@@ -229,15 +229,16 @@ $module: 'listing';
 			top: 0;
 			height: 100%;
 			width: 100%;
-			overflow-x: hidden;
-			overflow-y: scroll;
 			padding: $nav-offset 0 20px 0;
 			z-index: $middle;
 		}
 	}
 }
 
 .#{$module}--detailed {
+	overflow-y: scroll;
```

Comment:
if you move this here, it needs to also be on --saved. There's a block with both selectors you could move this to